src/tests/test_ggplot.py

```diff
@@ -409,6 +415,20 @@ def test_facet_wrap_stacked_histogram_cmap(diamonds_data):
     )
 
 
+@_cleanup_cm()
+@image_comparison(
+    baseline_images=["test_facet_wrap_default_with_nulls"],
+    extensions=["png"],
+    remove_text=False,
+)
+def test_facet_wrap_default_with_nulls(penguins_data):
+    (
```

Comment:
did u try with the dummy data instead of penguins?

Reply:
yes, I tried using the dummy data as well. Worked fine

Reply:
Better to generate with some dummy data containing nulls instead of penguins data